### ISSUE TITLE
Syncing does not work in Firefox

### ISSUE THREAD

**Issue by sbstn87** [alexpdraper/reading-list#88]
Hi,

I am using Firefox 62.0.3 and unfortunately the syncing feature does not work. Otherwise the extension would be perfect!

Best regards,
Sebastian

**Comment by sbstn87** [comment]
Unfortunately yes. I have it opened at the moment on my PC and my phone and the syncing does not work. I tried it also on a second PC and links I have added to the reading list did not show up there. What is more, the extension itself did not sync to my second PC and I had to install it manually.

Now I have updated Firefox to version 63 and that did not help.

**Comment by alexpdraper** [comment]
Are you signed in to the same Firefox account on both computers? If you go to `about:preferences#sync` and look at your sync settings, do you have “Add-ons” checked?

**Comment by sbstn87** [comment]
Of course. Reading List is the only extension I have installed that did not sync between my PCs. Just a few minutes ago two extensions which I have installed previously on a different PC have show up here. So that's probably not the case of faulty sync feature of Firefox.

**Comment by alexpdraper** [comment]
Weeeeeeeeird. I’m not sure how to replicate the problem here. Maybe try exporting your existing reading list and then re-installing it. It’s extra weird that the extension had to be manually installed on the second PC. If that’s an issue we have control over then I don’t even know where to start. I’ll try to look into it some more and see if I can figure anything out.

**Comment by sbstn87** [comment]
I have reinstalled it on my PC and smartphone. After adding new links and syncing devices it still does not work. When I have access to the other PC I will check if the links show up there. And if the extension itself will sync there.

By the way - I do not know how the extension sync between PCs, but I suppose that using bookmarks to do it would solve the issue.

**Comment by sbstn87** [comment]
OK, I have checked it again on the second PC and the issue remains. The extension itself did not sync and after installing it manually, the saved links did not show up.
